test/Zapper.ts

```diff
@@ -119,6 +125,14 @@ async function getDAIToUser (r: ContractTestContext) {
     const tokenDAI = new ethers.Contract(daiAddress, abiUSDTToken, owner);
     await router.swapExactETHForTokens(bnFromNum(500, 18), [addressWETH9, daiAddress], owner.address, 1670978314, { value: bnFromNum(1) });
 }
+
+async function getOUSDToUser (r: ContractTestContext) {
```

Comment:
Are you sure this is working? I think OUSD does not have a good uniswap pool so would probably fail